# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xa/hotfix-install.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xa-2.4.1/Makefile.vanilla	2024-09-29 18:58:46.160000000 +0100
+++ xa-2.4.1/Makefile	2024-09-29 18:58:11.907000000 +0100
@@ -29,7 +29,7 @@
 
 TESTS=ALL
 
-all: killxa xa uncpk
+all: xa uncpk
 
 killxa:
 	rm -f xa
